scripts/python/lib/ipmi.py

```diff
@@ -109,7 +109,7 @@ def ipmi_fru2dict(fru_str):
                 line = split[0] + ': "' + split[1] + '"'
                 yaml_data.append(line)
     yaml_data = '\n'.join(yaml_data)
-    return yaml.load(yaml_data)
+    return yaml.full_load(yaml_data)
```

Comment:
Will full_load also read !vault keyword (used in the ansible vault)? That was the reason why we stayed at 3.13 last time.

Reply:
You're right, `yaml.full_load()` does not come with a `!vault` constructor. This is the part that escaped my memory - we already wrote a [YAMLVault](https://github.com/open-power-ref-design-toolkit/power-up/blob/1512aeb40b67277b822a9a394c9f2e6483e5025f/software/wmla120.py#L1875-L1887) class to address this. With all of this PyYAML loader business I didn't realize we need to specify the Loader class when calling `add_constructor()` (see 1512aeb).